elide-integration-tests/src/test/java/example/Author.java

```diff
@@ -11,6 +11,7 @@
 import com.yahoo.elide.annotation.Paginate;
 import com.yahoo.elide.annotation.SharePermission;
 
+import lombok.Builder;
```

Comment:
Is this unused?

Reply:
@aklish Strange. This was not there on my branch. Anyway it's gone now

Reply:
This is still there.